backend/api/Profiles/Parcel/ParcelEvaluationProfile.cs

```diff
@@ -19,7 +19,7 @@ public ParcelEvaluationProfile()
                 .IncludeBase<Entity.BaseEntity, Pims.Api.Models.BaseModel>();
 
             CreateMap<Model.ParcelEvaluationModel, Entity.ParcelEvaluation>()
-                .ForMember(dest => dest.ParcelId, opt => opt.Ignore())
+                .ForMember(dest => dest.ParcelId, opt => opt.MapFrom(src => src.PropertyId))
```

Comment:
Odd to keep the property name as `PropertyId` for the `Parcel`?

Reply:
Yeah, the FE treats building evaluations and parcel evaluations as the same object, which uses property id instead of parcel/building id. The actual type is inferred based on the object structure.